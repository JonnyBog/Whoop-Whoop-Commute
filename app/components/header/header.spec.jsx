import React from 'react';
import renderer from 'react-test-renderer';

import Header from 'components/header/header';

describe('Components', () => {
  describe('Header', () => {
    it('should export', () => {
      expect(Header).toBeDefined();
    });

    /**
     * createRenderTree
     * @returns {JSX} component tree
     */
    function createRenderTree () {
      return renderer.create(
        <Header />
      ).toJSON();
    }

    describe('Appearance', () => {
      it('should render', () => {
        const tree = createRenderTree();
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
