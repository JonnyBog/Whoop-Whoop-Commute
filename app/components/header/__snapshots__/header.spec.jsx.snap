// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Header Appearance should render 1`] = `
<div
  className="sc-bwzfXH liBCIH sc-bdVaJa ldinxp sc-bdVaJa ldinxp"
>
  <div
    className="sc-bdVaJa jwmcyC"
  >
    <h1
      className="header__Title-sc-17ykvny-0 XOKSO"
    >
      <div
        className="header__TitleWhoop-sc-17ykvny-1 joDEFi"
      >
        Wh
        <span
          className="header__TitleWhoopOs-sc-17ykvny-2 jCPFPg"
        >
          oo
        </span>
        p Wh
        <span
          className="header__TitleWhoopOs-sc-17ykvny-2 jCPFPg"
        >
          oo
        </span>
        p
      </div>
      <div
        className="header__TitleCommute-sc-17ykvny-3 jtkWfn"
      >
        Commute
      </div>
    </h1>
    <p
      className="header__Message-sc-17ykvny-4 iuekhx"
    >
      This is a development app. Git repo 
      <a
        href="https://github.com/JonnyBog/Whoop-Whoop-Commute"
      >
        here
      </a>
    </p>
  </div>
</div>
`;
